guides/slack.md

```diff
@@ -0,0 +1,67 @@
+
+# Profiles
+
+Please complete your profile fully on joining slack - this means a suitable profile picture (it does not have to be a picture of you, but a clear picture of your face is ideal), your full name and a representative display name.
+
+You can optionally fill in the pronunciation guide for your name and add your preferred pronouns if you feel comfortable doing so.
+
+Please also include your up to date email address and group/role, and the location you are based.
+
+# Channels
+
+Slack channels broadly fall under the following categories:-
+
+ - Project channels (named after the project). If you are working on that delivery, it is expected you'll be in that channel, otherwise open project channels are optional.
+ - Supply channels (prefixed `supply-`). These channels are used by groups focusing on how we _supply_ deliverables. Optional unless directly involved with this group.
+ - Demand channels (prefixed `demand-`). These channels are used by groups focusing on how we engage with other organisations. Optional unless directly involved with this group.
+ - Group channels (such as `ops-` or `academy-`). Channels used by 
+ - Specific purpose channels (usually prefixed with their temporary purpose, for example `bid-`). Temporary channels used for teams fulfilling a specific related task. Optional unless involved with the task, and will be archived after completion.
+ - General channels (no prefix). Company wide chanels such as `announcements`, `help` or regional channels such as `manchester` and _community of practice_ channels prefixed with `cop-`, these channels are used for cross company communication and tend have channel specific usage expecations, which should be detailed in their channel topic.
+ - Off topic channels (prefixed `x-`). These channels are entirely optional to join and use, feel free to create off topic channels if your interest or hobby is not yet represented!
+ - Bot channels or 'firehose' channels. It is occasionally useful to setup bots that post information to specific channels periodically, it is advised that you only need to join these channels if you are involved in monitoring such information.
+ 
+## Mandatory channels
+
+As of writing, it is expected every slack user will be present in `#announcements`, this is so that everyone can be reached in case of an important announcement.
+There are no other channels mandatory for _all_ users, but it is expected you will be in channels relevant to your working group.
+
+## Private channels and group chats
+
+Often private channels will be set of for specific tasks and group chats may be used to have small private discussions if necessary. It's worth noting that everything posted on our slack is logged and could be subject to a Freedom of Information request (specifically for channels relating to public sector projects), so it is wise to always use slack as if your message could be read.
```

Comment:
* Private channels can be used for specific tasks and group chats may be used to have small private discussions if necessary